packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -99,12 +104,52 @@ export default class VscodeHatRenderer {
     return this.decorationMap[hatStyle];
   }
 
+  private async updateHatsDirWatcher() {
+    this.destroyHatsDirWatcher();
+
+    const hatsDir = vscode.workspace
+      .getConfiguration("cursorless.experimental")
+      .get<string>("hatShapesDir")!;
+
+    if (hatsDir && fs.existsSync(hatsDir)) {
```

Comment:
I would be tempted to throw an error if they've set it but it doesn't exist. Confusing to just silently do nothing. See snippets impl for how we're doing that there

Reply:
Agreed